lib/src/main/java/com/orange/ods/compose/component/card/OdsCardTitleFirst.kt

```diff
@@ -96,8 +94,8 @@ fun OdsCardTitleFirst(
                     .padding(horizontal = dimensionResource(id = R.dimen.spacing_m)),
                 verticalAlignment = Alignment.CenterVertically
             ) {
-                thumbnailRes?.let { OdsImageCircleShape(painter = painterResource(id = it)) }
-                Column(modifier = Modifier.padding(start = thumbnailRes?.let { dimensionResource(id = R.dimen.spacing_s) }
+                thumbnail?.let { OdsImageCircleShape(painter = it) }
+                Column(modifier = Modifier.padding(start = thumbnail?.let { dimensionResource(id = R.dimen.spacing_s) }
                     ?: run { 0.dp })) {
```

Comment:
This is not the subject of this PR, but we can use `orElse` instead of `?: run`.